ksql-common/src/test/java/io/confluent/ksql/reactive/ReactiveSubscriberTest.java

```diff
@@ -87,7 +86,7 @@ public void cancel() {
     assertThatEventually(afterSubScribeCalled::get, is(true));
     subscriber.onNext("record0");
     subscriber.onComplete();
-    awaitLatch(latch);
+    assertThat(latch.await(2000, TimeUnit.MILLISECONDS), is(true));
```

Comment:
nit: this can just be `awaitLatch(latch)`.

Reply:
ack